argobots.spec

```diff
@@ -109,6 +115,10 @@ rm -f %{buildroot}%{_libdir}/*.{l,}a
 %doc README
 
 %changelog
+* Wed Sep 04 2024 Brian J. Murrell <brian.murrell@intel.com> - 1.2-1
+- Update to 1.2
+- Add patch 411e5b3 fixing DAOS-14248
```

Comment:
Could you please update the datestamp here and add a comment about the 2nd patch that has been added and it's ticket number.

Please also feel free to change the author here to yourself since you did the most significant amount of work.

Reply:
- [ ] Add reference to the new patch [bb0c908](https://github.com/pmodels/argobots/commit/bb0c908abfac4bfe37852eee621930634183c6aa) fixing configuration issue

Reply:
Fixed with commit b740042

Reply:
> * [x]  Add reference to the new patch [bb0c908](https://github.com/pmodels/argobots/commit/bb0c908abfac4bfe37852eee621930634183c6aa) fixing configuration issue
> * [x]  Update author and timestamp

Fixed with commit b740042